iceoryx_posh/test/moduletests/test_mq_interface.cpp

```diff
@@ -280,6 +298,18 @@ TEST_F(CMqInterface_test, MqBase_IsInitialized)
     CMqInterface_IsInitialized<MqBase>(base);
 }
 
+TEST_F(CMqInterface_test, DestroyMessageQueueReturnsHasErrorFalse)
```

Comment:
Test name inconsistent with what is happening in the test.

Test name does not reflect what is happening, it calls ``base.closeMessageQueue()`` ...
I do not know whether it is the correct expected behavior for a constructed MqBase to return true when closeMessageQueue is called without any other action before. It appears to be happening though and does not look wrong per se. 
But the test name is not really expression the test intention.

Reply:
I think if we call EXPECTs directly in the test, the name will experssion the test intention